src/bsoncxx/string/view_or_value.cpp

```diff
@@ -15,20 +15,26 @@
 #include <bsoncxx/config/prelude.hpp>
 
 #include <bsoncxx/string/view_or_value.hpp>
+#include <bsoncxx/exception/error_code.hpp>
```

Comment:
Ah, no, I had thought about trying to make `data()` throw when appropriate but realized there was no good way to do that.